src/gtk/animate.cpp

```diff
@@ -466,4 +446,36 @@ void wxAnimationCtrl::OnTimer(wxTimerEvent& WXUNUSED(ev))
     }
 }
 
+
+// static
+wxAnimationImpl* wxAnimationCtrl::CreateAnimationImpl(wxAnimationImplType implType)
+{
+    switch (implType) {
+        case wxANIMATION_IMPL_TYPE_GENERIC:
+            return new wxAnimationGenericImpl();
+
+        case wxANIMATION_IMPL_TYPE_NATIVE:
+            return new wxAnimationGTKImpl();
+
+        default:
+            return NULL;
```

Comment:
Minor but it would be better to remove `default:` and move `return NULL` below the switch.

The rationale is that the compiler would then give us a warning here if we ever add a new element to the enum over which we switch, while it won't if there is a default case.